collector/replication_slots.go

```diff
@@ -70,23 +73,31 @@ func (PGReplicationSlotCollector) Update(ctx context.Context, db *sql.DB, ch cha
 		var wal_lsn int64
 		var flush_lsn int64
 		var is_active bool
+		var is_active_value int
+
 		if err := rows.Scan(&slot_name, &wal_lsn, &flush_lsn, &is_active); err != nil {
 			return err
 		}
 
+		if is_active {
+			is_active_value = 1
+		} else {
+			is_active_value = 0
+		}
```

Comment:
Go style nit.

Also, I just noticed that we have a bunch of snake case in this codebase. Will have to fix that up separately.

```suggestion
		isActiveValue := 0
		if is_active {
			isActiveValue = 1
		}
```